docs/uml-class-diagrams/display02/production/DisplayCommManagerImplementation/DisplayCommManagerImplementation.puml

```diff
@@ -2,12 +2,15 @@
 
 package edu.oakland.production.display02 {
     class DisplayCommManagerImplementation {
+        MiddlewareCommInterface middleware
         __
+        {abstract} + DisplayCommManagerImplementation(MiddlewareCommInterface middleware)
         .. Use Case 1 ..
-
-        .. Use Case 2 ..
+        {abstract} + TrackData evaluateRfidRequest(int rfidNum)
+        {abstract} + TrackData passRequestToMiddlewareCommInterface(int rfidNum)
+        {abstract} + void receiveRequestFromDisplay2WayInterface(int rfidNum)
```

Comment:
Remember: you're implementing this here, so it shouldn't be abstract. Also, are these private? If so, they should have the minus sign instead of the + sign.